.github/workflows/docker-publish.yml

```diff
@@ -9,6 +9,9 @@ on:
 jobs:
   docker:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
```

Comment:
is this permission is required?

Reply:
yes in order to write/upload the package (image)